packages/app/src/cli/models/extensions/specifications/editor_extension_collection.ts

```diff
@@ -0,0 +1,49 @@
+import {BaseSchema} from '../schemas.js'
+import {createExtensionSpecification} from '../specification.js'
+import {zod} from '@shopify/cli-kit/node/schema'
+
+export interface InCollection {
+  handle: string
+}
+
+const InCollectionSchema = zod.object({
+  handle: zod.string(),
+})
+
+export const EditorExtensionCollectionSchema = BaseSchema.extend({
+  in_collection: zod.union([zod.array(InCollectionSchema), zod.array(zod.string())]).optional(),
+  type: zod.literal('editor_extension_collection'),
+})
+
+function isArrayofStrings(arr: string[] | InCollection[]) {
+  return arr.every((item) => typeof item === 'string')
+}
+
+const editorExtensionCollectionSpecification = createExtensionSpecification({
+  identifier: 'editor_extension_collection',
+  schema: EditorExtensionCollectionSchema,
+  appModuleFeatures: (_) => ['bundling'],
```

Comment:
validation and localization to follow in other PRs

Reply:
I'm pretty sure you don't need `bundling` here, can you try to remove it?

Reply:
Ah yes, just removed it! Thanks